### PR TITLE
docs:  update fpga tools in page https://opae.github.io

### PR DESCRIPTION
**Pull request by anandaravuri**
-add fpga tools to  https://opae.github.io/

Signed-off-by: anandaravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: doc/sphinx/conf.py.in [hunk 1]
```diff
@@ -168,10 +168,12 @@ man_pages = [
...
-    ("docs/fpga_tools/host_exerciser/host_exerciser.md", 'host_exerciser', u'Host Exerciser lpbk and memory', [author], 8),
-    ("docs/fpga_tools/hssi_ethernet/hssistats.md", 'hssistats', u'High Speed Serial Interface ethernet statistics', [author], 8),
-    ("docs/fpga_tools/hssi_ethernet/hssimac.md", 'hssimac', u'High Speed Serial Interface ethernet max', [author], 8),
-    ("docs/fpga_tools/hssi_ethernet/hssiloopback.md", 'hssiloopback', u'High Speed Serial Interface ethernet loopback', [author], 8),
+    ("docs/fpga_tools/host_exerciser/host_exerciser", 'host_exerciser', u'Host Exerciser lpbk and memory', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssistats", 'hssistats', u'High Speed Serial Interface ethernet statistics', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssimac", 'hssimac', u'High Speed Serial Interface ethernet max', [author], 8),
+    ("docs/fpga_tools/hssi_ethernet/hssiloopback", 'hssiloopback', u'High Speed Serial Interface ethernet loopback', [author], 8),
+    ("docs/fpga_tools/opaeuio/opaeuio", 'opaeuio', u'Binding and Unbinding of a DFL device', [author], 8),
```
Comment (tswhison) [inline doc/sphinx/conf.py.in:175]: This tool (opaeuio script) no longer exists.  Please remove it.

File: doc/sphinx/index.rst.in [hunk 2]
```diff
@@ -63,4 +62,10 @@ The main documentation for the site is organized into following sections:
    docs/fpga_tools/opaevfio/opaevfio
    docs/fpga_tools/pci_device/pci_device
    docs/fpga_tools/opae.io/opae.io
-   
+   docs/fpga_tools/host_exerciser/host_exerciser
+   docs/fpga_tools/hssi_ethernet/hssistats
+   docs/fpga_tools/hssi_ethernet/hssimac
+   docs/fpga_tools/hssi_ethernet/hssiloopback
+   docs/fpga_tools/opaeuio/opaeuio
```
Comment (tswhison) [inline doc/sphinx/index.rst.in:69]: This tool (opaeuio script) no longer exists.  Please remove it.

File: doc/sphinx/conf.py.in (hunk 1 above)
Comment (anandaravuri) [reply doc/sphinx/conf.py.in]: removed opaeuio

File: doc/sphinx/index.rst.in (hunk 2 above)
Comment (anandaravuri) [reply doc/sphinx/index.rst.in]: removed opaeuio

**Review by tswhison — APPROVED** [review]

**Pull request merged**